style.css

```diff
@@ -482,27 +582,36 @@ img {vertical-align: middle;}
 	width: 80%;
 	position: absolute;
 	color: #555c66;
-    margin: 0px 10% 0px 10%;
+    margin: 0px 18% 0px 10%;
 	padding: 110px 0px 0px 0px;
 }
 
 /* What to bring images & text & tips (index) */
 
+.what_to_bring-container{
+    grid-row: 4;
+	grid-column-start: 1;
+	grid-column-end: 15;	
+	background: rgb(221,147,20);

```

Comment:
I might want to change the background colour to something else, it isn't as appealing as I previously though, (orange background)